user-guide/query-examples-hacks/redash-hacks.md

```diff
@@ -81,5 +82,13 @@ CASE
         WHEN '{{date}}' = 'All' THEN date IS NOT NULL
         ELSE date = '{{date}}'
   END
-  ```
+```
 Kelly (thanks Kelly!) from Hudl shared this in our [Slack community](http://slack.redash.io).
+
+## Adding Tags to Queries {#query-tags}
+
+Somewhere in the middle between naming queries with long-tediou-very descriptive names and having fancy clickable tags we neutrally present to you - #tags!
+
+Add a #marketing, #devops or #xmas-sales "tag" to your query names and then search for all queries with that #tag.
+
+Same goes for Dashboards (although you can already group them by topics when using a `marketing:online-traffic`, `marketing:new-vs-returning-users` convention). Read more [here](../dashboards/dashboards.md#grouping_dashs).
```

Comment:
👍 let's just change the dashboard examples to `Marketing: Online Traffic` and `Makreting: New vs. Returning Users`.